TACC/Core-Portal
# bug/FP-1572: Fix UI overflow

## Overview: ##

- Fixes UI overflow on the dashboard and manage account views caused by custom css-column widths.
- (Unrelated) Adds the `lonestar6` system to the system mapping settings

## Related Jira tickets: ##

* [FP-1572](https://jira.tacc.utexas.edu/browse/FP-1572)

## Summary of Changes: ##

## Testing Steps: ##
1. Go to https://cep.dev/workbench/dashboard and https://cep.dev/workbench/account
2. Confirm you can't scroll horizontally to the right. (See **before** pic below for reference

## UI Photos:

- Before:
    <img width="1792" alt="Screen Shot 2022-03-23 at 3 14 12 PM" src="https://user-images.githubusercontent.com/20326896/163853823-a4b3fcb1-94b8-4a09-9027-0c1f687a13fe.png">


## Notes: ##
As a follow-up, we should think about converting these Section components to use css grid
